qemu/tests/qmp_command.py

```diff
@@ -262,10 +245,6 @@ def send_cmd(cmd):
     cmd_return_value = params.get("cmd_return_value")
     exception_list = params.get("exception_list", "")
 
-    # HOOKs
-    if result_check == 'qmp_cpu':
-        pre_cmd = "cpu index=%d" % (int(params['smp']) - 1)
-
```

Comment:
Agree with @yanan-fu , current qmp_cpu_check() function can be implemented by check_result(), it is not specific any more when removing old checking method.